components/brave_rewards/core/wallet_provider/connect_external_wallet.h

```diff
@@ -58,6 +66,7 @@ class ConnectExternalWallet {
   void CheckLinkageCallback(endpoints::GetWallet::Result&& result);
 
   base::RetainingOneShotTimer linkage_checker_;
+  OAuthInfo oauth_info_;
```

Comment:
I think this should be `protected`, and then `Authorize()` wouldn't need the `const OAuthInfo& oauth_info` parameter.

Reply:
Nor `GetOAuthLoginURL()`.